pkgs/by-name/pl/plasmusic-toolbar/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+  pname = "plasmusic-toolbar";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "ccatterina";
+    repo = "plasmusic-toolbar";
+    rev = "v${version}";
+    hash = "sha256-yc6hUi5tICpG3SacYnWVApYQXPN4Yrw6+BFd9ghlqxA=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/plasma/plasmoids/plasmusic-toolbar
+    cp -r $src/src/* $out/share/plasma/plasmoids/plasmusic-toolbar
```

It'd be more classic to use:

```suggestion
    cp -r src/* $out/share/plasma/plasmoids/plasmusic-toolbar
```

Since you are already at `$src` which is also writable.

---

I completely forgot about it